index.html

```diff
@@ -121,7 +121,28 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+        <div class="card">
+          <p class="name">Winkel Yin</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/Winkel515" target="_blank">Winkel515</a>
+          </p>
+          <p class="about">Trying to get back into coding</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://leetcode.com/Winkel515/" target="_blank" title="First Resource">Leetcode</a>
```

Comment:
Since the URL that you provided here redirects to your profile, you can add it in the `contact` section above. 
Also, add any other helpful resource in here.